Doc/reference/datamodel.rst

```diff
@@ -510,6 +510,10 @@ Callable types
       | :attr:`__closure__`     | ``None`` or a tuple of cells  | Read-only |
       |                         | that contain bindings for the |           |
       |                         | function's free variables.    |           |
+      |                         | The ``cell_contents``         |           |
+      |                         | attribute can be used for     |           |
+      |                         | used for reading and writing  |           |
```

Comment:
Duplicated "used for".

It may be better to describe the cell object somewhere else (for example in a footnote or in a separate paragraph following this table) than in a narrow table cell.